### ISSUE TITLE
[$15]Register and submit button should be announced, even though disabled

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3249]
**Describe the bug**

Register and submit button should be announced, even though disabled

**To Reproduce**
1. Go to sample challenge page
2. Ensure that the screen reader is switched on. 
3. Try to tab onto Register or submit button (Be sure not to be previously registered)

**Expected behavior**
Announce buttons, but announce as "Disabled" when they are disabled.

**Actual behavior**
Does not receive focus

**Additional context**

Type: Buttons

Page: https://www.topcoder.com/challenges/30098248

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099940 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099940 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099940 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
Fixed by PR https://github.com/topcoder-platform/topcoder-react-utils/pull/49

**Comment by veshu** [comment]
@tosha5252 @drasticdpk @lakshmiathreya Could you please verify this?

**Comment by drasticdpk** [comment]
@veshu  issue has reference with https://github.com/topcoder-platform/community-app/issues/3237 it seems code has not merge yet.
@r0hit-gupta Here 'submit' is having link instead of button  have you fixed that one too?
![image](https://user-images.githubusercontent.com/25297859/64315809-cebbb280-cfd2-11e9-8510-1b2ab56e1ae3.png)

**Comment by lakshmiathreya** [comment]
Disabled Register and Submit buttons continue to be skipped. Pls clarify intended fix for this Issue. @veshu @drasticdpk @sushilshinde

**Comment by sushilshinde** [comment]
topcoder-react-utils dependecies

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

React Utils Dependency

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099940<br/><br/>```This is an automated message for crazyk via Topcoder X```
